### PR TITLE
Longest common subsequence

### PR DESCRIPTION
**Pull request by Nikola-Mircic**
An algorithm for finding longest common subsequence of two strings using dynamic programming.

### CODE REVIEW

**Comment by Nikola-Mircic** [comment]
I'm not quite sure why are there commits for difference array. Please notify me if that is a problem, so I can fix it.

File: src/strings/lcs.jl
Comment (Panquesito7) [inline src/strings/lcs.jl:10]:
```suggestion
# Contributed by:- [Nikola Mircic](https://github.com/Nikola-Mircic)
```

**Review by Panquesito7 — DISMISSED** [review]
Overall, everything's looking great! 🚀 Thank you! 😄👍

**Review by uncomfyhalomacro — APPROVED** [review]

**Comment by uncomfyhalomacro** [comment]
@Panquesito7 Seems CI is still showing version 1.6. Hmm I guess I can wait for you to merge this since I do not have write access.

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**
